pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+// This file contains a collection of methods that can be used from go-resful to
+// generate Swagger API documentation for its models. Please read this PR for more
+// information on the implementation: https://github.com/emicklei/go-restful/pull/215
+//
+// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
+// are on one line! For multiple line or blocks that you want to ignore use ---.
+// Any context after a --- is ignored.
+//
+// Those methods can be generated by using hack/update-generated-swagger-docs.sh
+
+// AUTO-GENERATED FUNCTIONS START HERE
+func (APIVersion) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "An APIVersion represents a single concrete version of an object model.",
+		"name":     "Name of this version (e.g. 'v1').",
+		"apiGroup": "The API group to add this object into. Default 'experimental'.",
+	}
+}
+
+func (AWSElasticBlockStoreVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "AWSElasticBlockStoreVolumeSource represents a Persistent Disk resource in AWS.\n\nAn Amazon Elastic Block Store (EBS) must already be created, formatted, and reside in the same AWS zone as the kubelet before it can be mounted. Note: Amazon EBS volumes can be mounted to only one instance at a time.",
+		"volumeID":  "Unique ID of the persistent disk resource in AWS (Amazon EBS volume). More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+		"fsType":    "Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+		"partition": "The partition in the volume that you want to mount. If omitted, the default is to mount by volume name. Examples: For volume /dev/sda1, you specify the partition as \"1\". Similarly, the volume partition for /dev/sda is \"0\" (or you can leave the property empty).",
+		"readOnly":  "Specify \"true\" to force and set the ReadOnly property in VolumeMounts to \"true\". If omitted, the default is \"false\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+	}
+}
+
+func (Binding) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "Binding ties one object to another. For example, a pod is bound to a node by a scheduler.",
+		"metadata": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
```

Comment:
I think i found the answer to my question about standard vs. non-standard. In our current API it links to "[ObjectMeta](http://kubernetes.io/v1.0/docs/api-reference/definitions.html#_v1_objectmeta)" (which we should ensure is also included in the new output we ill be generating). And here is doesnt distinguish between standard/non-standard, simply stating "an object". 

I think qualifying the object with "standard" is confusing and if its unnecessary, we should remove it. 

That said, its still unclear what to specify for this property. Is it the ObjectMeta:name? In other words, how to you specify an Object's metadata? We need to add an example here.

The text that I originally proposed was:
"Metadata of the standard object that you want to bind...."

Removing "standard", it should be:
"Metadata of the object that you want to bind...."

But how is the metadata specified? If its specified by the name, then it would be:
"The name of the metadata for the object that you want to bind. ...."